src/generators/app/templates/src/server/i18n.js

```diff
@@ -0,0 +1,34 @@
+import i18n from 'i18next';
+import Backend from 'i18next-node-fs-backend';
+import { LanguageDetector } from 'i18next-express-middleware';
+
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
+  .init({
+    detection: {
+      order: ['cookie'],
+      caches: false
+    },
+    backend: {
+      loadPath: '../shared/i18n/locales/{{lng}}/{{ns}}.json',
+    },
+    lng: 'en',
+    fallbackLng: 'en',
+    // have a common namespace used around the full app
+    ns: ['common'],
+    defaultNS: 'common',
+    debug: process.env.NODE_ENV === 'development',
+
+    interpolation: {
+      escapeValue: false, // not needed for react!!
+      formatSeparator: ',',
+      format: function(value, format, lng) {
+        if (format === 'uppercase') return value.toUpperCase();
+        return value;
+      }
+    }
+  });
+
```

Comment:
We should move this to `shared/lib/i18n.js`. it can be a function that along the lines of:

```
import i18next from 'i18next';

export default (middlewares, init)=>{
  let i18n = i18next;
  middlewares.forEach(middleware => i18n = i18n.use(middleware));
  i18n.init(Object.assign({
    lng: 'en',
    fallbackLng: 'en',
    // have a common namespace used around the full app
    ns: ['common'],
    defaultNS: 'common',
    debug: config.get('i18n.debug'),

    interpolation: {
      escapeValue: false, // not needed for react!!
      formatSeparator: ',',
      format: function(value, format, lng) {
        if (format === 'uppercase') return value.toUpperCase();
        return value;
      }
    }
  }, init));
  return i18n;
}

```